sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -125,7 +125,7 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
 
   createQueryTest("SPARK-8976 Wrong Result for Rollup #1",
     """
-      SELECT count(*) AS cnt, key % 5,GROUPING__ID FROM src group by key%5 WITH ROLLUP
```

Comment:
Since we already have the results, just copy them as golden files (or copy them into test cases).